Patch by Robert Scheck <robert@fedoraproject.org> for libunicap <= 0.9.12 to declare the
struct as extern in header files (https://gcc.gnu.org/gcc-10/porting_to.html).

--- libunicap-0.9.12/common/queue.h			2010-01-17 08:49:29.000000000 +0100
+++ libunicap-0.9.12/common/queue.h.gcc10		2020-02-09 23:30:16.963136320 +0100
@@ -24,7 +24,7 @@
 #include <sys/time.h>
 #include <semaphore.h>
 
-struct _unicap_queue
+extern struct _unicap_queue
 {
       sem_t sema;
       sem_t *psema;
--- libunicap-0.9.12/cpi/euvccam/queue.h		2010-01-19 07:52:49.000000000 +0100
+++ libunicap-0.9.12/cpi/euvccam/queue.h.gcc10		2020-02-09 23:30:05.037048713 +0100
@@ -24,7 +24,7 @@
 #include <sys/time.h>
 #include <semaphore.h>
 
-struct _unicap_queue
+extern struct _unicap_queue
 {
 		sem_t sema;
 		sem_t *psema;
